search-provider/eks-discovery-feed-provider.c

```diff
@@ -435,6 +469,166 @@ handle_article_card_descriptions (EksDiscoveryFeedDatabaseContentProvider *skele
 }
 
 static void
+get_word_of_the_day_content_cb (GObject *source,
+                                GAsyncResult *result,
+                                gpointer user_data)
+{
+  EkncEngine *engine = EKNC_ENGINE (source);
+  DiscoveryFeedQueryState *state = user_data;
+
+  g_application_release (g_application_get_default ());
+
+  GError *error = NULL;
+  g_autoptr(GSList) models = NULL;
+  GSList *shards = NULL;
+
+  if (!models_and_shards_for_result (engine,
+                                     state->provider->application_id,
+                                     result,
+                                     &models,
+                                     &shards,
```

Comment:
Ah right ... I guess we only care about loading the shards into memory so that we can query them with Eknc. In that case, maybe a separate func makes sense.